### PR TITLE
Change encoding enforced by Jabref.

### PR DESCRIPTION
**Pull request by jppelteret**
The encoding is changed from ISO-8859-1 ( https://en.wikipedia.org/wiki/ISO/IEC_8859-1 ), which supports some special characters, to ASCII ( https://en.wikipedia.org/wiki/ASCII ), which does not. The intent is that Jabref will not allow files with special characters to be saved if the encoding does not allow it. This should help with detection if some anomalous characters slip into an entry (e.g. if entries are downloaded from DOI, e-print, etc.).

I understand that this is a bit hostile towards our European colleagues, but I think it will help ensure that all entries are rendered as intended. All special characters will thus need to be explicitly formed with tex commands. If it makes a difference, we already enforce this when `offline/publication_list.tex` is built.

### CODE REVIEW

**Comment by jppelteret** [comment]
The `bib` files for years 2014, 2015 and 2017 appear to contain some incompatible characters... Before this is merged I want to work out where they are so I can replace them.

**Comment by jppelteret** [comment]
Note to self: Jabref is not very helpful about emitting where illegal characters can be found. When you save it just complains. This command is useful to show which line in a file cannot be converted from one format to another:
```sh
iconv --from-code=iso-8859-1 --to-code=us-ascii publications-xxxx.bib
```

File: publications-2015.bib [hunk 1]
```diff
@@ -267,7 +267,7 @@ @article{2015_24
 }
 
 @article{2015_25,
-   author  = {T. Carraro and C. Goll and A. Marciniak-Czochra and A. Mikelic´},
```
Comment (davydden) [inline publications-2015.bib]: i think this is supposed to be just `\'{c}`, see https://arxiv.org/abs/1401.6278

Comment (davydden) [reply publications-2015.bib]: but to be fair, i don' know how `\'{c}'` renders...

File: publications-2014.bib [hunk 2]
```diff
@@ -1175,7 +1175,7 @@ @article{2014_109
 }
 
 @article{2014_110,
-   author  = {A. Vidal-Ferrandiza and R. Fayeza and D. Ginestarb and G. Verdúa},
+   author  = {A. Vidal-Ferrandiza and R. Fayeza and D. Ginestarb and G. Verd\'{u}a},
```
Comment (davydden) [inline publications-2014.bib]: apparently, same `Verdu`, see https://www.sciencedirect.com/science/article/pii/S0306454914002473

Comment (davydden) [reply publications-2014.bib]: oh, all surnames have extra `a`/`b` here

File: publications-2015.bib (hunk 1 above)
Comment (jppelteret) [reply publications-2015.bib]: Oh yeah, thats a mistake... My editor placed an extra character there. Will fix.

File: publications-2014.bib (hunk 2 above)
Comment (jppelteret) [reply publications-2014.bib]: Indeed :-)

**Comment by jppelteret** [comment]
@davydden Thanks for pointing out those issues. I think that I've fixed them now.

File: publications-2014.bib (hunk 2 above)
Comment (bangerth) [reply publications-2014.bib]: I bet they were copied verbatim from a list of authors where a footnote symbol referred to an address at the bottom of the page. Might have been my fault.

Comment (jppelteret) [reply publications-2014.bib]: Yip, I agree that this is probably what happened. Although it wasn't a big deal to fix the issue now, I don't think that we should worry too much about this right now. We'll have to go through all of the entries to double check the data anyway.

**Pull request merged**